src/ts/lib/storage.ts

```diff
@@ -58,6 +58,35 @@ class Storage {
 			this.storage[key] = JSON.stringify(o);
 		};
 	};
+
+	unsetPath (key: string, path: string[]): void {
```

Comment:
This is used to remove a key at a path in a stored object (in the case of this PR remove the last opened object indexed by the window id).